common.sh

```diff
@@ -39,14 +39,20 @@ NODES_PLATFORM=${NODES_PLATFORM:-"libvirt"}
 MASTER_NODES_FILE=${MASTER_NODES_FILE:-"ocp/master_nodes.json"}
 
 export RHCOS_IMAGE_URL=${RHCOS_IMAGE_URL:-"https://releases-rhcos.svc.ci.openshift.org/storage/releases/maipo/"}
-export RHCOS_IMAGE_VERSION="${RHCOS_IMAGE_VERSION:-47.284}"
-export RHCOS_IMAGE_NAME="redhat-coreos-maipo-${RHCOS_IMAGE_VERSION}"
-# FIXME(shardy) - we need to download the -openstack as its needed
-# for the baremetal nodes so we get config drive support,
-# or perhaps a completely new image?
+export RHCOS_LATEST="$(curl ${RHCOS_IMAGE_URL}/builds.json | jq -r ".builds[0]")"
+export RHCOS_IMAGE_VERSION="${RHCOS_IMAGE_VERSION:-${RHCOS_LATEST}}"
+export RHCOS_IMAGE_FILENAME_OPENSTACK_GZ="$(curl ${RHCOS_IMAGE_URL}/${RHCOS_IMAGE_VERSION}/meta.json | jq -r '.images.openstack.path')"
+export RHCOS_IMAGE_NAME=$(echo $RHCOS_IMAGE_FILENAME_OPENSTACK_GZ | sed -e 's/-openstack.*//')
+# FIXME(shardy) note the -openstack image doesn't work for libvirt
+# as the qemu ignition config injection described in the docs at
+# https://coreos.com/os/docs/latest/booting-with-libvirt.html
+# doesn't work - probably we need to download both as the
+# -openstack one may be needed for the baremetal nodes so we get
+# config drive support, or perhaps a completely new image?
+export RHCOS_IMAGE_FILENAME="${RHCOS_IMAGE_NAME}-qemu.qcow2"
```

Comment:
The -qemu image is no longer needed ref https://github.com/openshift-metalkube/dev-scripts/pull/226/commits/2967acaa701c0dfc5fe44deb656a7e793418ecbc

Reply:
Removed that, thanks